src/Microsoft.Extensions.Configuration.AzureAppConfiguration/ConfigurationClientManager.cs

```diff
@@ -21,8 +24,13 @@ namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
     /// there won't be multiple threads calling this client at the same time.
     /// </remarks>
     internal class ConfigurationClientManager : IConfigurationClientManager
-    {
-        private readonly IList<ConfigurationClientWrapper> _clients;
+    {
+        private IList<ConfigurationClientWrapper> _clients;
+
+        private IEnumerable<string> _connectionStrings;
+        private IEnumerable<Uri> _endpoints;
+        private TokenCredential _credential;
+        private ConfigurationClientOptions _clientOptions;
 
         public ConfigurationClientManager(IEnumerable<string> connectionStrings, ConfigurationClientOptions clientOptions)
```

Comment:
Is there a reason why you left in the original 2 constructors? It seems like you replaced all uses of them with the new constructor where `AzureAppConfigurationOptions` is passed in.